core/src/main/resources/error/error-classes.json

```diff
@@ -714,6 +714,11 @@
     ],
     "sqlState" : "22023"
   },
+  "INVALID_FUNCTION_ARGS" : {
```

Comment:
How about to introduce sub-classes of `WRONG_NUM_ARGS`:
- WITHOUT_SUGGESTION
- WITH_SUGGESTION

And declare in the common message template that
```
Invalid number of arguments for the function <funcName>.
```